Python/initconfig.c

```diff
@@ -64,9 +64,9 @@ Options (and corresponding environment variables):\n\
 -X opt : set implementation-specific option\n\
 --check-hash-based-pycs always|default|never:\n\
          control how Python invalidates hash-based .pyc files\n\
---help-env      : print help about Python environment variables and exit\n\
---help-xoptions : print help about implementation-specific -X options and exit\n\
---help-all      : print complete help information and exit\n\
+--help-env: print help about Python environment variables and exit\n\
+--help-xoptions: print help about implementation-specific -X options and exit\n\
+--help-all: print complete help information and exit\n\
```

Comment:
The previous spacing was following the style of spacing for short options (see lines just above)

Reply:
Well it was my intent when I added them!  Maybe you disagree on the result 🙂 
When I say «following», I meant that all short options have colons at the same column, check-hash-based-pycs is special and uses a new line (with the help text aligned to the help texts for short options), and this new group of long options has colons at the same column (but not the same at the short options, that doesn’t work).